schematools/types.py

```diff
@@ -302,6 +354,18 @@ def is_nested_table(self) -> bool:
         """
         return (
             self.get("type") == "array"
+            and self.nm_relation is None
+            and self.get("items", {}).get("type") == "object"
+        )
+
+    @property
+    def is_through_table(self) -> bool:
+        """
+        Checks if field is a possible through table.
+        """
+        return (
+            self.get("type") == "array"
+            and self.nm_relation is not None
             and self.get("items", {}).get("type") == "object"
```

Comment:
Misschien is het beter om `and self.get("items", {}).get("type") == "object"` naar de `nm_relation` functie te verplaatsen?